pom.xml

```diff
@@ -45,6 +45,9 @@
     <jetty.perf-helper.version>1.0.6</jetty.perf-helper.version>
     <ant.version>1.10.9</ant.version>
     <unix.socket.tmp></unix.socket.tmp>
+    <!-- OSGI import-package -->
+    <osgi.common.import.packages>org.slf4j;version="[1.7,3.0)", org.slf4j.event;version="[1.7,3.0)", org.slf4j.helpers;version="[1.7,3.0)", org.slf4j.spi;version="[1.7,3.0)"</osgi.common.import.packages>
```

Comment:
This property should be renamed to `slf4j.import.packages`.

Reply:
I considered that, as that is how it is currently used.
But using the more generic `osgi.common.*` prefix means we can add to this list for other reasons without having to touch all of the locations that use this new property again.

Reply:
How about `osgi.slf4j.import.packages`?
I don't what to drop the `osgi` portion.

Reply:
Changed to `osgi.slf4j.import.packages` instead in commit 4720f07cc7d376fa6b0b374f0832736f88bdfadc